app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
+    <string name="app_name">RefShoppe</string>
```

Comment:
I left a similar content on @HaroldFS's PR related to naming. We should be consistent with how we name our reference apps. Let's circle up today after standup.